src/test.ts

```diff
@@ -74,6 +75,7 @@ export function runTest(testDirectory: string, rulesDirectory?: string | string[
     const results: TestResult = { directory: testDirectory, results: {} };
 
     for (const fileToLint of filesToLint) {
+        const startTime = new Date().getTime();
```

Comment:
Since we are running in a node environment, what about using `process.hrtime()`?